### PR TITLE
Remove constant for Gemm and Add node for A if A is an constant in Gemm

### PR DESCRIPTION
**Pull request by edis219**
1. Remove constant for Gemm 
2. Add node for A if A is an constant in Gemm

### CODE REVIEW

File: onnx_coreml/_operators_nd.py [hunk 1]
```diff
@@ -674,11 +674,21 @@ def _convert_gemm(builder, node, graph, err):
     transB = node.attrs.get('transB', False)
 
     A = node.inputs[0]
+    if A in node.input_tensors:
+        A_tensor = node.input_tensors[A]
+        builder.add_load_constant_nd(
+            name=node.name + "_const_" + A,
+            output_name='const_' + A,
```
[the remark this answers is not in the document]
Comment (edis219) [reply onnx_coreml/_operators_nd.py]: done according to your comment

File: onnx_coreml/_transformers.py [hunk 2]
```diff
@@ -760,6 +760,22 @@ def __call__(self, graph):  # type: (Graph) -> Graph
                     axes = node.attrs.get('axes', None)
                     output = np.squeeze(x, axis = tuple(axes)) 
                 transformation_performed = True
+            elif node.op_type == 'Gemm':
+                print("Constant remover on Gemm : %s" % node.name)
```
[the remark this answers is not in the document]
Comment (edis219) [reply onnx_coreml/_transformers.py]: done according to your comment

**Comment by edis219** [comment]
Re-work according to the comments, please review again.

**Comment by bhushan23** [comment]
@edis219 could you please sign the license/cla?

**Comment by edis219** [comment]
@bhushan23  The license/cla has been signed. Thanks a lot!

**Pull request merged**